front/app/utils/testUtils/rtl.tsx

```diff
@@ -10,16 +10,14 @@ import { LiveAnnouncer } from 'react-aria-live';
 
 const AllTheProviders = ({ children }) => {
   return (
-    <div id="modal-portal">
```

Comment:
Doesn't this makes for a comeback in modal warnings ?

Reply:
I was still seeing the warnings and tried to fix by moving this around. It didn't work though, not sure why...
Was planning to look deeper into this issue later on but if you have an idea of why the warning appears and how to fix, do let me know :)